spec/support/providers.rb

```diff
@@ -15,7 +15,10 @@
   before :each do
     allow(FbGraph2::User).to receive(:me).and_return social_user_auth
     allow(Twitter::REST::Client).to receive(:new).and_return social_user_auth
-    allow(GooglePlus::Person).to receive(:get).and_return social_user_auth
     allow(Instagram).to receive(:client).and_return social_user_auth
+
+    stub_request(:get, /https:\/\/www\.googleapis\.com\/oauth2\/v3\/tokeninfo\?id_token=*/)
```

Comment:
1: =* is "Any number of equal signs". Simply leaving it off is sufficient, since this regexp doesn't match to the end of the string.
2: This can be made less ugly: %r{https://www\.googleapis\.com/oauth2/v3/tokeninfo\?id_token=*}
3: Consider adding a \A at the beginning of the regexp.